cmake/Modules/FindAWSSDK_EP.cmake

```diff
@@ -126,4 +126,10 @@ if (AWSSDK_FOUND)
     endif()
 
   endforeach ()
+
+  # the SDK does not include links to some transitive dependencies
+  if (TILEDB_AWSSDK_EP_BUILT AND WIN32)
+    list(APPEND AWS_EXTRA_LIBRARIES userenv ws2_32 Wininet
```

Comment:
Just noting that some of these we link here: https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/CMakeLists.txt#L362

Reply:
Thanks. I moved the ones for AWSSDK here, and used the existing logic in `tiledb/CMakeLists.txt`.